client/components/Scratchpad.js

```diff
@@ -31,7 +32,24 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const getToolCursor = (tool, scale) => {
+const distance = (from, to) => {
+  const dx = Math.abs(from.x - to.x);
```

Comment:
nit: Math.abs is unnecessary since you're just squaring it anyway

Reply:
Ha :-) I though about that after sending the CL, will fix it 👍